src/components/shared/card.tsx

```diff
@@ -207,6 +210,18 @@ const iconStyles = (format: Format): SerializedStyles => {
     `;
 }
 
+const commentIconStyle = (): SerializedStyles => {
+    return css`
+        width: 2.0rem;
+        height: 1.4375rem;
+        display: inline-block;
+        fill: #E05E00;
```

Comment:
Can we use the opinion colour from the design system? you had it as `fill: ${opinion[400]};` before

Reply:
Ah yes your right, i took from storybook when adjusting for size of speech mark and must've pasted raw clor